qemu/tests/block_copy.py

```diff
@@ -412,19 +412,33 @@ def create_file(self, file_name):
         """
         params = self.params
         session = self.get_session()
-        file_create_cmd = params.get("create_cmd", "touch FILE")
+        file_create_cmd = params.get("create_command", "touch FILE")
         test_exists_cmd = params.get("test_exists_cmd", "test -f FILE")
         if session.cmd_status(test_exists_cmd.replace("FILE", file_name)):
-            session.cmd(file_create_cmd.replace("FILE", file_name))
-        session.cmd("md5sum %s > %s.md5" % (file_name, file_name))
+            session.cmd(file_create_cmd.replace("FILE", file_name), timeout=200)
+        session.cmd("md5sum %s > %s.md5" % (file_name, file_name), timeout=200)
+        if params["os_type"] == "linux":
+            session.cmd("sync && echo 3 > /proc/sys/vm/drop_caches")
```

Comment:
Both Linux and Windows all need to sync date here.  In our local win_utils, there has one "Sync/sync64.exe", "Sync/sync.exe"  to do this.

We can add  different  "sync_command"  in cfg for  Linux and Windows,  and call it here.